tomcat/tomcat-jta/src/main/java/org/jboss/narayana/tomcat/jta/NarayanaJtaServletContextListener.java

```diff
@@ -91,6 +116,9 @@ public void contextDestroyed(ServletContextEvent servletContextEvent) {
         });
     }
 
+    /**
+     * If node identifier wasn't set byt property manager, then set default.
```

Comment:
typo (byt). Can you {@link DEFAULT_NODE_IDENTIFIER} for the default value.

Similarly for the other default values below.